src/Microsoft.DocAsCode.Dfm/Tokens/DfmFencesToken.cs

```diff
@@ -19,6 +19,12 @@ public DfmFencesToken(IMarkdownRule rule, IMarkdownContext context, string name,
             SourceInfo = sourceInfo;
         }
 
+        public DfmFencesToken(IMarkdownRule rule, IMarkdownContext context, string name, string path, SourceInfo sourceInfo, string lang, string title, IDfmFencesBlockPathQueryOption pathQueryOption, string queryStringAndFragment)
+            : this(rule, context, name, path, sourceInfo, lang, title, pathQueryOption)
```

Comment:
change above method to: 
        public DfmFencesToken(IMarkdownRule rule, IMarkdownContext context, string name, string path, SourceInfo sourceInfo, string lang, string title, IDfmFencesBlockPathQueryOption pathQueryOption)
            : this(rule, context, name, path, sourceInfo, lang, title, pathQueryOption, null) ?